### ISSUE TITLE
Missing PySide2 in dependencies?

### ISSUE THREAD

**Issue by kinow** [CaptainDario/DaKanjiRecognizer#4]
Hi,

After creating a blank virtual environment (Py 3.8, Ubuntu LTS 20.04), and modifying the tflite runtime as per #3 , I installed it with ` pip install -r requirements_rel.txt`.

Then I cloned the extra dependency with `git clone https://github.com/CaptainDario/ETL_data_reader.git` and installed it with `cd ETL_data_reader/; cd ETL_data_reader/; cd ..`.

However, when I tried running `src/main.py`, it failed with the following message:

```
$ python src/main.py 
Traceback (most recent call last):
  File "src/main.py", line 5, in <module>
    from PySide2.QtGui import QIcon
ModuleNotFoundError: No module named 'PySide2'
```

The application started after I installed it with

```
$ pip install PySide2
Collecting PySide2
  Downloading PySide2-5.15.2-5.15.2-cp35.cp36.cp37.cp38.cp39-abi3-manylinux1_x86_64.whl (164.3 MB)
     |████████████████████████████████| 164.3 MB 4.9 kB/s 
Collecting shiboken2==5.15.2
  Downloading shiboken2-5.15.2-5.15.2-cp35.cp36.cp37.cp38.cp39-abi3-manylinux1_x86_64.whl (956 kB)
     |████████████████████████████████| 956 kB 1.4 MB/s 
Installing collected packages: shiboken2, PySide2
Successfully installed PySide2-5.15.2 shiboken2-5.15.2
```

So I wonder if that's missing from the requirements.txt file, or if I missed an installation step?

Thanks!
Bruno

**Comment by CaptainDario** [comment]
Hi Kinow,

you are right PySide2 is missing.
I checked just now with a separate venv and I will update the requirements now.

Thank you for your support

**Comment by kinow** [comment]
Thank **you** for sharing such amazing app @CaptainDario !

**Comment by CaptainDario** [comment]
I updated the dependencies in a [new branch](https://github.com/CaptainDario/DaKanjiRecognizer/tree/Fix-dependencies_build).
There I also added in the README the platform dependent installation of tflite-runtime.

Does this work for your setup too?

**Comment by kinow** [comment]
@CaptainDario looks good to me! You moved the platform-specific dependency out of the requirements file, and added instructions on how to install it separately on each platform. That should work fine :+1: 

Thanks!

**Comment by CaptainDario** [comment]
Fixed by this pull request #7 .
